CODEOWNERS

```diff
@@ -0,0 +1 @@
+/** @prestodb/committers
```

Comment:
What's supposed to be the `/**`?

Reply:
It means all files recursively from the root.  It's intended to give committers access to everything.